creator-node/src/apiHelpers.ts

```diff
@@ -135,32 +141,56 @@ export const sendResponse = (
   res.status(resp.statusCode).send(resp.object)
 }
 
+/**
+ * Wrapper helper method to send response with metrics involved
+ * @param {Object} req Express request object
+ * @param {Object} res Express response object
+ * @param {Object} routeResponse error or success response object
+ * @param {Object} prometheusResult prometheus results to track
+ */
+export const sendResponseWithMetric = (
+  req: CustomRequest,
+  res: Response,
+  routeResponse: ApiResponse,
+  prometheusResult: { result: string; mode?: string }
```

Comment:
how do you know the type of this object? it is meant to be generic right? e.g. diff route metrics can have different schema for this, def can't expect them all to specify a `mode`

Reply:
@SidSethi that's why mode is optional

Reply:
yup, basically what johannes said. the ? implies optional, so if there is no mode defined, that is ok. but there is always a result (as i've seen most prom metrics have result in them)